package.json

```diff
@@ -26,12 +26,14 @@
     "postinstall": "husky install",
     "precommit": "lint-staged",
     "test": "yarn workspace @se-2/hardhat test",
-    "vercel": "yarn workspace @se-2/nextjs vercel",
-    "vercel:yolo": "yarn workspace @se-2/nextjs vercel:yolo"
+    "vercel": "vercel"
```

Comment:
I think the problem with `yarn vercel` was originated here, the old vercel scripts dissapeared during the migration.

Reply:
Thanks, you're right! Now it works. Confused `package.json`s 😄